src/vfocus.js

```diff
@@ -300,3 +300,13 @@ VFocus.prototype.children = function(){
 VFocus.prototype.addChild = function(child){
   this.vNode.children.push(child);
 };
+
+VFocus.prototype.indexOf = function(focus){
+  //if we are passing a VFocus use the vNode
+  var vNode = (focus.vNode || focus);
+  return this.children().indexOf(vNode);
+}
+
+VFocus.prototype.splice = function(){
+  return this.children().splice.apply(this, arguments);
```

Comment:
`addChild` adds a child to the focus. This `splice` splices the children of the vFocus. I found that a bit confusing.

But more importantly I'm not sure about proxying array methods and putting them on `VFocus`. Would rather keep `VFocus` cohesive. Otherwise, where do we draw the line? If we'd like to access a lower level API such as the one for arrays then there's nothing wrong about accessing the raw objects (even if it makes that bit of code slightly longer).